### DISCUSSION TITLE
Display more information and images from Wikipedia and Wikimedia Commons

### DISCUSSION THREAD

**Discussion by vachan-maker** [organicmaps/organicmaps#1879] [Ideas]
It would be really helpful if Organic Maps could display images, description and more information from the wikidata tag that is provided. OSMAnd already has this feature and shows images from wikimedia commons. 

Organic Maps only displays the URL of the wikipedia page. It would be cool, if the application could display more information like a short description, etc.

**Comment by biodranik** [comment]
I thought that we already have such issue or discussion, no?

**Reply by vachan-maker** [reply]
Sorry, I did not search here properly. 
Yes, this is the discussion #1052
Closing this
Thanks.

**Comment by vachan-maker** [comment]
Looks like these discussions cannot be closed. Please continue on #1052
